projects/1.16/assets/charcoal_pit/lang/zh_cn.json

```
@@ -0,0 +1,247 @@
+  "effect.charcoal_pit.poison_resistance": "毒抗性",
+  "effect.charcoal_pit.wither_resistance": "凋零抗性",
+  "advancements.charcoal_pit.root.title": "木炭坑",
+  "advancements.charcoal_pit.root.description": "嗨",
+  "advancements.charcoal_pit.fire_starter.title": "猴子生火!",
+  "advancements.charcoal_pit.fire_starter.description": "在有打火石之前制作一个起火器来生火",
+  "advancements.charcoal_pit.log_pile.title": "许多方块组成的方块?",
+  "advancements.charcoal_pit.log_pile.description": "制作一个原木堆",
+  "advancements.charcoal_pit.charcoal.title": "精炼木头",
+  "advancements.charcoal_pit.charcoal.description": "从木炭坑中获取木炭",
+  "advancements.charcoal_pit.aeternalis.title": "比Irene的还好",
+  "advancements.charcoal_pit.aeternalis.description": "如果你足够幸运，或者足够执着，就能在末地城里找到永恒遗迹",
+  "advancements.charcoal_pit.straw.title": "干草大王",
+  "advancements.charcoal_pit.straw.description": "用锄头把高草砍倒",
+  "advancements.charcoal_pit.brick.title": "砖，砖，砖，砖，砖，砖，砖",
+  "advancements.charcoal_pit.brick.description": "用稻草和圆木烧制陶窑砖",
+  "advancements.charcoal_pit.coke.title": "焦口可乐",
+  "advancements.charcoal_pit.coke.description": "用砖块建造一个焦炉，将煤块加工成焦炭",
+  "advancements.charcoal_pit.clay_pot.title": "你就是哈利·波特",
+  "advancements.charcoal_pit.clay_pot.description": "将粘土制成罐用于储存或矿石加工",
+  "advancements.charcoal_pit.ceramic_pot.title": "为了存放你的绿宝石",
+  "advancements.charcoal_pit.ceramic_pot.description": "从陶窑中取出一个陶罐",
+  "advancements.charcoal_pit.copper.title": "铜石并用时代",
+  "advancements.charcoal_pit.copper.description": "把矿石放在陶土器皿里，放在窑里烧，让它熔化",
+  "advancements.charcoal_pit.bloomery.title": "方坯",
+  "advancements.charcoal_pit.bloomery.description": "用砖块建造一个锻铁炉，加上一个风口和风箱来冶炼铁",
+  "advancements.charcoal_pit.iron.title": "铁",
+  "advancements.charcoal_pit.iron.description": "在锻铁炉里放置铁矿石和煤（或木炭）并点燃，用风箱鼓风进行熔炼",
+  "advancements.charcoal_pit.creosote.title": "提取器",
+  "advancements.charcoal_pit.creosote.description": "使用杂酚油漏斗从木炭坑和焦炉收集杂酚油",
+  "advancements.charcoal_pit.barrel.title": "就像大金刚一样",
+  "advancements.charcoal_pit.barrel.description": "制作一个桶来储存液体",
+  "advancements.charcoal_pit.alcohol.title": "手工酿酒厂",
+  "advancements.charcoal_pit.alcohol.description": "酿造12种酒精饮料中的任何一种",
+  "advancements.charcoal_pit.diggy_diggy_hole.title": "Diggy Diggy Hole",
+  "advancements.charcoal_pit.diggy_diggy_hole.description": "酿造蜂蜜Dewois，矮人的长生不老药",
+  "advancements.charcoal_pit.vinegar.title": "RCOOH",
+  "advancements.charcoal_pit.vinegar.description": "将水果加入酒精中制成醋",
+  "advancements.charcoal_pit.cheese.title": "Gromit,奶酪!棒极了!",
+  "advancements.charcoal_pit.cheese.description": "把醋加到牛奶里做成奶酪",
+  "advancements.charcoal_pit.wine_conosur.title": "葡萄酒鉴赏家",
+  "advancements.charcoal_pit.wine_conosur.description": "酿造所有类型的酒精",
+  "advancements.charcoal_pit.all_effects.title": "酒精!",
+  "advancements.charcoal_pit.all_effects.description": "同时获得所有酒类的药水效果",
+  "advancements.charcoal_pit.kernels.title": "陈腔滥调",
+  "advancements.charcoal_pit.kernels.description": "只有食物标签",
+  "advancements.charcoal_pit.fugu.title": "什么可能出错?",
```

|Feedback Priority|
|---|
|🧗‍♀️ Boulder / **Blocking**|

```suggestion
  "advancements.charcoal_pit.fugu.title": "啥东西可能会出错？",
```